storage/cloudspanner/log_storage.go

```diff
@@ -839,16 +839,16 @@ func (tx *logTX) GetLeavesByIndex(ctx context.Context, indices []int64) ([]*tril
 	// TODO: replace with INNER JOIN when spannertest supports JOINs
 	// https://github.com/googleapis/google-cloud-go/tree/master/spanner/spannertest
 	stmt := spanner.NewStatement(
-		`SELECT 
+		`SELECT
```

Comment:
This might be harder than it's worth the effort, but I would like to see these trailing whitespace changes in a separate change. I'd be very sad if this changed any functionality (!), but my cautious side needs to speak out.

Reply:
done (I think/hope)